types.ts

```diff
@@ -0,0 +1,61 @@
+// faucet-pipeline-core types
+export interface FaucetPlugin<T> {
+		(config: T[], assetManager: AssetManager, options: FaucetPluginOptions): FaucetPluginFunc
```

Comment:
Is double indentation here intentional?

Reply:
Nope!

Reply:
Fixed in 877c5250a12fa5aa22c7ce305fae442228c3f522